pages/onboarding.jsx

```diff
@@ -7,12 +7,20 @@ const Onboarding = () => {
 
   const handleRedTeamChange = (index, field, value) => {
     const updatedRedTeamPlayers = [...redTeamPlayers];
+    if(field === 'idNumber' && value !== '' && !/^\d*$/.test(value)) {
+      console.log("Invalid ID Number");
+      return;
+    }
```

Comment:
It appears that the input tag's type="number" attribute is preventing non-numbers from being typed. So the code isn't able to reach here.

I don't think it's high priority, but I'm thinking maybe there is a way to add a red outline like you see with some forms when this action is attempted that is tied to the input tag emitting an event.

Reply:
This apparently depends on the browser. So I'll merge this change then.